lib/auth/style.scss

```diff
@@ -63,6 +64,22 @@
     font-weight: 400;
   }
 
+  input:focus {
+    border-color: $studio-simplenote-blue-50;
+    box-shadow: 0 0 0 1px $studio-simplenote-blue-50;
+    outline: none;
+  }
+
+  input:disabled {
+    border-color: $studio-gray-5;
+    color: $studio-gray-20;
+  }
+
+  input:read-only {
+    border-color: $studio-gray-10;
+    background-color: rgba(0, 0, 0, 0.05);
+  }
```

Comment:
I saw that it's possible to have both `focus` and `read-only` states applied at the same time. In this case the `read-only` styles have preference over the `focus` ones because they're located after and therefore the inner border is set to grey. 

If we want to keep the same styles (only have blue border), we should put this part before `input:focus` and the same in the code below for the dark theme.

Here you can check how it looks like when a `readonly` input is focused:
<img width="1077" alt="Screenshot 2021-04-14 at 10 05 52" src="https://user-images.githubusercontent.com/14905380/114676426-92514c00-9d09-11eb-9dcd-e6be947215e9.png">

Reply:
Another good catch, thank you! I've updated this appropriately now.